woocommerce/woocommerce
# fix ESLint configuration regarding import path aliases

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.







### Changes proposed in this Pull Request:





While working on WooCommerce Admin packages, I noticed that ESLint raises a warning when path aliases are used. This PR fixes the warning.


https://github.com/woocommerce/woocommerce/assets/4463174/158c7117-dde9-4a70-9551-8ff4cc03f1e4




### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Ensure that the CI job regarding the lint doesn't fail.
2. Open `plugins/woocommerce-admin/client/customize-store/index.tsx` and ensure that any warning is visible in the import section.
3. Import a function using path aliases and ensure that ESLint doesn't throw a warning.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

Fix ESLint configuration regarding import path aliases.

#### Comment 

</details>
